huaweicloud/common/common.go

```diff
@@ -374,3 +374,20 @@ func GetEipsbyAddresses(client *golangsdk.ServiceClient, addresses []string, eps
 	}
 	return allEips, nil
 }
+
+// GetResourceIDs returns resource IDs from an order.
+func GetResourceIDs(client *golangsdk.ServiceClient, orderId string, onlyMainResource int) ([]string, error) {
```

Comment:
it's better to make the function name clearly.
```suggestion
func GetResourceIDsByOrder(client *golangsdk.ServiceClient, orderId string, onlyMainResource int) ([]string, error) {
```